tests/unit/utils/test_azurerm.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import saltext.azurerm.utils.azurerm_mod as azurerm_util
```

Comment:
**chore (blocking):** remove this file too, since we're removing the referenced module